core/src/test/java/feign/LoggerTest.java

```diff
@@ -76,16 +76,16 @@ public LogLevelEmitsTest(Level logLevel, List<String> expectedMessages) {
               "\\[SendsStuff#login\\] <--- HTTP/1.1 200 OK \\([0-9]+ms\\)")},
           {Level.HEADERS, Arrays.asList(
               "\\[SendsStuff#login\\] ---> POST http://localhost:[0-9]+/ HTTP/1.1",
-              "\\[SendsStuff#login\\] Content-Type: application/json",
               "\\[SendsStuff#login\\] Content-Length: 80",
+              "\\[SendsStuff#login\\] Content-Type: application/json",
```

Comment:
Changes here are not related to this pull request.  Please refrain from submitting changes that are purely formatting.

Reply:
Hello, @kdavisk6 @velo 

Firstly, thanks for the fast review.

Actually, I suppose these changes **do** relate on current commit: it's not just a formatting, but the point is the changing of header fields order to alphabetical.

I reverted only the **LoggerTest** file, ran "mvn clean install" once again locally and got following output:

https://gist.github.com/tworogue/3b9262457101f8753de01db3d9d4ba84

So it seems to me that these changes are mandatory in this particular case, they're not cosmetic and they really make difference.

What would you advice in regarding of fixing this (with no changing LoggerTest file)?

Reply:
Make sense, since your you are shorting the header keys, this file must be changed.

Thanks for the detailed explanation